.github/workflows/node.js.yml

```diff
@@ -13,6 +13,7 @@ env:
   REACT_APP_BEE_DEBUG_URL: http://localhost:1635
   NEXT_PUBLIC_BEE_DEBUG_URL: http://localhost:1635
   NEXT_PUBLIC_RPC_URL: https://xdai.dev.fairdatasociety.org
+  NEXT_PUBLIC_CONSENT_VIEWER: https://app.crviewer.dev.fairdatasociety.org
```

Comment:
this correct ? it will be deployed to main not dev

Reply:
This is dev configuration. The configuration for main is in the `.github/workflows/swarm.yml`